### PR TITLE
Content: Anro/Seekers Aliens

### PR DESCRIPTION
**Pull request by RisingLeaf**
----------------------
**Content (Artwork / Missions / Jobs)**

## Summary
This PR adds two new alien species to the north west of the galaxy. These are the Seekers and the Anro.
Both species are about tier 2.5 and when the player first meets them, they are on hostile terms with each other.
They are not originally from the milky way but from the black eye galaxy which can be visited after some story
progress.
The Seekers will be the first alien species who are a swarm intelligence while the Anro are just another species.
<details>
<summary>Map Changes</summary>
North-West:
 Map Changes
Black Eye:
 Map Changes
</details>
<details>
<summary>History of both species</summary>
Both the Seekers and the Anro come from the Black-Eye galaxy. While the Anro around the black hole in the center of the galaxy the Seekers lived in the region of space that we would call the core. The Black-Eye galaxy has one speciality: Around the black hole there is an extreme dense gas flow rotating against the rotation direction of the whole galaxy and around the black hole are rotating a lot of giant stars of all classes(This is an scientific fact). That's why the Anro were not able to get out of their zone for a long time.
<br></br>
Meanwhile the Seekers discovered space travel with hyperdrives but when they encountered the other species they were shocked by the wars they fighting against each other. As a swarm intelligence they had never before seen a war or something like that. They decided that the wars had to stop and they developed weapons for the first time in their history. Because they were superior to other species through their immense intelligence they were soon able to dominate all the other species. 
After that a period of ultimate peace was following. They forbid all other species to build weapons but still allowed them to interact with each other.
<br></br>
During that time the Anro discovered the jump drive and they were finally able to break out of their own territory. When they first came out of their space the seekers in their haughtiness wanted to  subdue them just like the other species of the galaxy. But the Anro were stronger than they thought and this careless action of the Seekers provoked a war of a class that has never been seen before in the galaxy. Both species fought with all their might. While the Anro were slightly higher advanced the Seekers had more resources.
<br></br>
The war was going on and the Seekers managed to build Jump Drives and they copied some of the gravitation manipulating tech of the Anro. But then the Anro discovered how to cut of Hyperspace-Links. With that technology they cut of links that connected parts of the galaxy to far away to jump between them. The territory of the Seekers was getting smaller and smaller and with the territory their resources were getting smaller. The Anro were able to slowly treat them back until the Seekers saw their last refuge in going through an wormhole in the outer space of the galaxy that directly brought them to the milky way. From there they were able to hold the Anro back because of their long supply lines.
In this state the player finds them when he first gets there.
<br></br>
</details>
<details>
<summary>Description of the Seekers</summary>
The Seekers
First of all "nomen est omen": The name "Seekers" is a short summary of their goals. The Seekers seek for the knowledge of everything. As a swarm intelligence their mind power is far more powerfull than most of the other species.
To their look: They are somewhat like ants but then their skin is like the one of a snake and their movements are smoother than the ones of an ant. They are a little bit smaller than a human and their physical strength equals the strength of a good trained human. Each of them lives about 500 years.
<br></br>
They communicate over electromagnetic fields therefore the player needs a special device to be able to talk to them. To be able to work as a swarm even in space they have small electromagnetic senders in their ship. But even though they are a swarm each of them has a small own personality. This lead to something you could call a split personality, even if it is far more complex, in the past.
<br></br>
Their ships are all named after mystical creatures. Expressing the unknown things in the universe. Their ship design is held simple meaning that they have only one hull layer and shield systems integrated.
</details>
<details>
<summary>Description of the Anro</summary>
The Anro
The name of the Anro comes out of my fantasy so it has not really a meaning. But i think it is a nice sounding name.
<br></br>
The Anro are a strong species with a high resistance to gravity or radiation because their star is surrounding a  near black hole. Although they are strong their psychic abilities should not be underestimated. They have a good eye for connectivity between things and can advance in a short time if they are under pressure.
<br></br>
They are  humanoid even though they are a lot smaller than the average human. Their extremities have 6 joints, one for each rotation axis. They have exceptional good reflexes what makes them the perfect space pilots.
<br></br>
The Anro love peace and value it high because they had some bloody civil wars in their past. They also value freedom over all things.
</details>

## Todo
 - [ ] Most of the Story line
 - [ ] More ships for both sides
 - [ ] Balancing for outfits
 - [ ] outfit art
 - [ ] optional: create own effects for weapons
 - [ ] optional: create own sounds

## Artwork Checklist
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}

### CODE REVIEW

File: data/anro/planets.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+planet Arun
+	landscape land/water5
+	government "Anro"
+	description `This is a stormy world with a lot of water. The air is cool and smells like sea.`
+	spaceport `The Spaceport is small but it has all it needs.`
+
+planet Airum
+	landscape land/desert0
+	government "Anro"
+	description `This is the world where the Anro originally came from. There are large flat deserts and there is as good as no water around here. Big cities are standing out everywhere and a peacefull feeling lays in the air.`
```
Comment (TheMarksman-ES) [inline data/anro/planets.txt:10]:
```suggestion
	description `This is the world where the Anro originally came from. There are large flat deserts as far as the eye can see, and not a drop of water save the occasional mirage. Only the massive cities stand out within the ocean of sand, standing tall in defiance of the great pale sea.`
```

Comment (TheMarksman-ES) [reply data/anro/planets.txt:10]: Not going to do my usual proofreading run anytime soon, so that's all you're getting for now.

**Comment by EjoThims** [comment]
Excited for a half second about the placement at least, but this is North WEST, not North East.

Specifically it's additions to an already crowded area in a spot that has been blanket vetoed for the forseeable future.

Deep is being intentionally left isolated from new systems, for consistency in both galaxy sprite and starting lore on how they came to be there. The existing story just does not make sense if there are other aliens nearby, much less active, high powered threats.

**Comment by roadrunner56** [comment]
Are you on the ES community discord? If not, I would highly recommend joining. Usually PRs of this magnitude are discussed with others there before being submitted, in order to make sure that the concepts don't conflict with lore. Here's the link: https://discord.gg/ZeuASSx 
If you want to add a new species, check out https://github.com/endless-sky/endless-sky/pull/6438. It's an alien species that's being worked on by the main dev of the project. It's a good example of what's expected for large content PRs.

**Comment by thvk-net** [comment]
> Deep is being intentionally left isolated from new systems, for consistency in both galaxy sprite and starting lore on how they came to be there. The existing story just does not make sense if there are other aliens nearby, much less active, high powered threats.

Thats right but only if these aliens would be active and high powered threats.
I have to explain myself:
I originally didnt wanted to post the pull request today.

**Pull request closed without merging**